pages/docs/guides/pause-functions.mdx

```diff
@@ -0,0 +1,52 @@
+export const description = 'Learn how to pause an Inngest function.'
+
+# Pausing functions
+
+Inngest allows you to pause a function indefinitely. This is a powerful feature that can be useful if you are planning downtime for a database migration, for example, or if a user reports a bug and you want to stop processing events until you've fixed it.
+
+## When a function is paused
+
+It's important to understand what happens when a function is paused.
+
+**No data will be lost.** Inngest will continue receiving and storing your events, but the events won't trigger the paused function.
+
+**You can resume the function at any time.** No deployment or sync is required to un-pause your function.
+
+**Events received while the function was paused won't be reprocessed automatically** after you resume the function. You can use a [replay](/docs/platform/replay) to process events that were received while the function was paused.
+
+## How to pause a function
+
+Navigate to the function's page in Inngest Cloud, and click the Pause button near the top right corner:
+
+![Pause button in Inngest Cloud.](/assets/docs/guides/pause-functions/pause-btn.png)
+
+## Handling running invocations
+
+When you pause a function, you can choose how any currently-running invocations of that function should be handled:
+
+- "Pause immediately, then cancel after 7 days:" This is the default behavior. No further steps will be executed while the function is paused. If you resume the function within 7 days, these invocations will continue with the next step. Otherwise, they'll be cancelled.
```

Comment:
```suggestion
- **"Pause immediately, then cancel after 7 days:"** This is the default behavior. No further steps will be executed while the function is paused. If you resume the function within 7 days, these invocations will continue with the next step. Otherwise, they will be cancelled.
```